### PR TITLE
Collections forward menu

### PR DESCRIPTION
**Pull request by mikeger**
# Purpose of this PR

- Adds action menu on collection cells
- Forward
- Show In Conversation

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Forward.swift [hunk 1]
```diff
@@ -198,3 +198,35 @@ extension ConversationContentViewController: UIAdaptivePresentationControllerDel
...
+                self.messageWindow.moveDown(byMessages: UInt(indexInConversation - newestMessageIndexInMessageWindow))
+            }
+        }
+        
+        delay(0.01) {
+            let indexToShow = self.messageWindow.messages.index(of: messageToShow)
+            
+            let cellIndexPath = IndexPath(row: indexToShow, section: 0)
+            self.tableView.scrollToRow(at: cellIndexPath, at: .middle, animated: true)
+            delay(0.35) {
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Forward.swift]: And this

File: Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Forward.swift [hunk 2]
```diff
@@ -198,3 +198,35 @@ extension ConversationContentViewController: UIAdaptivePresentationControllerDel
...
+
+            if oldestMessageIndexInMessageWindow > indexInConversation {
+                self.messageWindow.moveUp(byMessages: UInt(oldestMessageIndexInMessageWindow - indexInConversation))
+            }
+            else {
+                self.messageWindow.moveDown(byMessages: UInt(indexInConversation - newestMessageIndexInMessageWindow))
+            }
+        }
+        
+        delay(0.01) {
```
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Forward.swift]: This basically does dispatch_async, to make sure the window has the time to expand. I will change the method implementation to wait until window actually shows the message.

File: Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Forward.swift (hunk 1 above)
Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Forward.swift]: Waiting for animation to finish (will be removed).

File: Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Forward.swift (hunk 2 above)
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Conversation/Content/ConversationContentViewController+Forward.swift]: Are you planning on doing that in this PR?

**Comment by mikeger** [comment]
@typfel I will update this PR, please do not merge yet

**Comment by mikeger** [comment]
@typfel actually I would disable menu in this PR and merge it, otherwise I would have a massive conflicts if I continue working on the feature. I will create the separate PR with fix for `scroll(to:)`

**Comment by mikeger** [comment]
@typfel updated, please review

**Pull request merged**